pkgs/applications/science/electronics/simulide/default.nix

```
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchbzr
+, libsForQt5
```

Now that the qt-type `callPackage` is used, the qt deps should be included like this.

```suggestion
, mkDerivation
, qmake
, qtserialport
, qtmultimedia
, qttools
, qtscript
```